server/src/entities/User.ts

```diff
@@ -27,5 +32,12 @@ export class User {
 
   @Property({type: 'text'})
   password!: string;
+
+  @Field()
+  @Property({type: 'text'})
+  userType!: string;
```

Comment:
Why is there user type and user role? What's the difference?

Reply:
ya they are the same, just one is use string as in put then one is user the list we provided. i forget remove the userType.